gulp/tasks/default.js

```diff
@@ -1,5 +1,10 @@
-(function() {
-    'use strict';
-    var gulp = require('gulp');
-    gulp.task('default', ['test', 'lint', 'jscs', 'coverage']);
-}());
+'use strict';
+
+var gulp = require('gulp');
+
+gulp.task('default', [
+    'doc-build',
+    'lint',
+    'jscs',
```

Comment:
I know this isn't related to the current PR, but I've seen this pattern now in a few of our repos (including Insights) where we separate "lint" from "jscs". I've always felt like they can be bundled together, e.g. "lint" calls `jshint` as well as `jscs`. If they're going to remain separate though, I think it's worth renaming `jscs` to `style` so that there's symmetry between `lint`/`style` - currently one of the task names represents a task to perform while the other is just the name of a tool.